.github/workflows/ci.yaml

```
@@ -9,8 +9,23 @@ on:
 
 jobs:
   pathogen-ci:
-    # Temporarily use this ref until the commit is merged into the main repo
-    # PR: https://github.com/nextstrain/.github/pull/62 (opened on 2023-09-27)
-    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@4f3074183d6ab612faed84444eefe08a5ded0c69
+    strategy:
+      matrix:
+        runtime: [docker, conda]
+    permissions:
+      id-token: write
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@master
+    secrets: inherit
     with:
-      build-dir: phylogenetic
+      runtime: ${{ matrix.runtime }}
+      run: |
+        nextstrain build \
+          phylogenetic \
+          --configfile profiles/ci/builds.yaml
```

I realized today that we don't _need_ the CI profile since we can copy over the example data here:
```suggestion
      run: |
        mkdir -p phylogenetic/data;
        cp -r -v phylogenetic/example_data/* phylogenetic/data/;
        nextstrain build phylogenetic
```
However, it's still nice to be able to run the example build with the single `nextstrain build` command, so I'm keeping it.